src/test/java/org/abstractj/kalium/fixture/TestVectors.java

```diff
@@ -105,4 +106,5 @@
     public static final String SIGN_SIGNATURE = "6bd710a368c1249923fc7a1610747403040f0cc30815a00f9ff548a896bbda0b" +
             "4eb2ca19ebcf917f0f34200a9edbad3901b64ab09cc5ef7b9bcc3c40c0ff7509";
     public static final String SIGN_PUBLIC = "77f48b59caeda77751ed138b0ec667ff50f8768c25d48309a8f386a2bad187fb";
+    public static final String SIGN_PUBLIC_PUBLIC = "35488a98f7ec26ae27099809afb27587b198b1197b5bcb0dec41153db2bf9952";
```

Comment:
What does PUBLIC_PUBLIC means? Maybe we should rename it to something like `PUBLIC_CURVE25519`?